### PR TITLE
Update sdk 2.8 and implement new webview sync

### PR DESCRIPTION
**Pull request by Quentin-Tran**

### CODE REVIEW

File: UMI3D-BROWSER/Assets/Dependencies/UMI3D SDK/Dependencies/Runtime/LoaderUtils/Runtime/MRTKShaderUtils.cs
Comment (RomainTirbisch) [inline UMI3D-BROWSER/Assets/Dependencies/UMI3D SDK/Dependencies/Runtime/LoaderUtils/Runtime/MRTKShaderUtils.cs:1]: Are comments expected here ?

File: UMI3D-BROWSER/Assets/Dependencies/UnityGeckoWebView/Sample/UnityGeckoWebViewSearch.cs
Comment (RomainTirbisch) [inline UMI3D-BROWSER/Assets/Dependencies/UnityGeckoWebView/Sample/UnityGeckoWebViewSearch.cs:1]: Header have been removed

File: UMI3D-BROWSER/Assets/Dependencies/UnityGeckoWebView/Scripts/UnityGeckoWebView.cs
Comment (RomainTirbisch) [inline UMI3D-BROWSER/Assets/Dependencies/UnityGeckoWebView/Scripts/UnityGeckoWebView.cs:1]: Header have been removed

File: UMI3D-BROWSER/Assets/Dependencies/UnityGeckoWebView/Scripts/UnityGeckoWebViewBounds.cs
Comment (RomainTirbisch) [inline UMI3D-BROWSER/Assets/Dependencies/UnityGeckoWebView/Scripts/UnityGeckoWebViewBounds.cs:1]: Header have been removed

File: UMI3D-BROWSER/Assets/Dependencies/UnityGeckoWebView/Scripts/UnityGeckoWebViewCallback.cs
Comment (RomainTirbisch) [inline UMI3D-BROWSER/Assets/Dependencies/UnityGeckoWebView/Scripts/UnityGeckoWebViewCallback.cs:1]: Header have been removed

File: UMI3D-BROWSER/Assets/Dependencies/UnityGeckoWebView/Scripts/UnityGeckoWebViewInput.cs
Comment (RomainTirbisch) [inline UMI3D-BROWSER/Assets/Dependencies/UnityGeckoWebView/Scripts/UnityGeckoWebViewInput.cs:1]: Header have been removed

**Review by RomainTirbisch — CHANGES_REQUESTED** [review]
Some header have been removed

File: UMI3D-BROWSER/Assets/Dependencies/UMI3D SDK/Dependencies/Runtime/LoaderUtils/Runtime/MRTKShaderUtils.cs
Comment (Quentin-Tran) [reply UMI3D-BROWSER/Assets/Dependencies/UMI3D SDK/Dependencies/Runtime/LoaderUtils/Runtime/MRTKShaderUtils.cs:1]: Yes

File: UMI3D-BROWSER/Assets/Dependencies/UnityGeckoWebView/Sample/UnityGeckoWebViewSearch.cs
Comment (Quentin-Tran) [reply UMI3D-BROWSER/Assets/Dependencies/UnityGeckoWebView/Sample/UnityGeckoWebViewSearch.cs:1]: "bien vu" as we say in French !

**Review by RomainTirbisch — APPROVED** [review]

**Pull request merged**